examples/rust/fft/README.md

```diff
@@ -0,0 +1,36 @@
+# Prerequisite
```

Comment:
Let's flesh this file out a little more.  Here's a good template to follow:

https://github.com/singlestore-labs/singlestoredb-extension-bloom-filters

Reply:
Updated